layout/net461/net461.depproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <NuGetTargetMoniker>.NETFramework,Version=v4.6.1</NuGetTargetMoniker>
```

Comment:
><NuGetTargetMoniker>.NETFramework,Version=v4.6.1</NuGetTargetMoniker> [](start = 4, length = 69)

Everything looks good in general, I just don't understand why are we using the net461 targeting pack instead of the net462.